samples/CommunityToolkit.Maui.Sample/Pages/Alerts/ToastPage.xaml.cs

```diff
@@ -0,0 +1,22 @@
+﻿using System.Reflection;
+using CommunityToolkit.Maui.Alerts;
+using CommunityToolkit.Maui.Core;
+using CommunityToolkit.Maui.Extensions;
+
+namespace CommunityToolkit.Maui.Sample.Pages.Alerts;
+
+public partial class ToastPage : BasePage
+{
+	IToast? toast;
```

Comment:
We probably don't need this field since we're re-initializing `IToast` in `ShowToastButtonClicked`